automation_tools/satellite6/upgrade/satellite.py

```diff
@@ -66,7 +66,7 @@ def satellite6_setup(os_version):
     # Set satellite hostname in fabric environment
     env['satellite_host'] = sat_host
     logger.info('Satellite {} is ready for Upgrade!'.format(sat_host))
-    if os.environ.get('RUN_EXISTANCE_TESTS'):
+    if os.environ.get('RUN_EXISTANCE_TESTS').lower() == 'true':
```

Comment:
Make sure to default to a string if the env var is not defined like:

```python
os.environ.get('RUN_EXISTANCE_TESTS', '')
```

Because if that env var is no defined it will return None (default value that get method returns) and calling `lower` on a None object raises and exception.